FluentScheduler/Scheduler/Schedule.cs

```diff
@@ -17,17 +17,68 @@ public class Schedule
         /// </summary>
         /// <param name="job">Job to be scheduled</param>
         /// <param name="cronExpression">The scheduling as a cron expression</param>
-        public Schedule(Action job, string cronExpression) =>
+        public Schedule(Action job, string cronExpression) : this(() => MakeAsync(job), cronExpression)
+        {
+        }

```

Comment:
Minor thing: rest of the project puts empty blocks on the same line (`Foo() { }`).